native/src/routes/news/containers/NewsContainer.js

```diff
@@ -75,15 +75,17 @@ const onRouteClose = (routeKey: string, dispatch: Dispatch<StoreActionType>) =>
 const refresh = (refreshProps: RefreshPropsType, dispatch: Dispatch<StoreActionType>) => {
   const { route, navigation, cityCode, language, newsId, selectedNewsType } = refreshProps
 
-  const navigateToNews = createNavigateToNews(dispatch, navigation)
-  navigateToNews({
+  const navigateTo = createNavigate(dispatch, navigation)
+  navigateTo({
+    route: NEWS_ROUTE,
     cityCode,
-    type: selectedNewsType,
-    language,
-    newsId,
-    forceRefresh: true,
-    key: route.key
-  })
+    newsType: selectedNewsType,
+    languageCode: language,
+    newsId: newsId || undefined
+  },
+  route.key,
+  true
+  )
```

Comment:
The formatting looks off for me here. I hink the `route.key` and `true` should be intended

Reply:
```
  navigateTo({
    route: POIS_ROUTE,
    cityCode,
    languageCode: language,
    cityContentPath: path || undefined
  }, route.key, true)
```

This formatting is used elsewhere in this PR.